src/Compilers/CSharp/Test/Syntax/Parsing/ParsingTests.cs

```diff
@@ -146,24 +141,18 @@ internal void UsingExpression(string text, params DiagnosticDescription[] expect
             UsingExpression(text, options: null, expectedErrors);
         }
 
-        /// <summary>
-        /// Parses given string and initializes a depth-first preorder enumerator.
-        /// </summary>
-        protected SyntaxTree UsingTree(string text, CSharpParseOptions? options = null)
+        protected SyntaxTree UsingTree(string text, params DiagnosticDescription[] expectedErrors)
         {
-            return UsingTree(text, options, verifyErrors: false, expectedErrors: null);
+            return UsingTree(text, options: null, expectedErrors);
         }
 
-        protected SyntaxTree UsingTree(string text, CSharpParseOptions? options, bool verifyErrors, DiagnosticDescription[]? expectedErrors)
+        protected SyntaxTree UsingTree(string text, CSharpParseOptions? options, params DiagnosticDescription[] expectedErrors)
         {
             VerifyEnumeratorConsumed();
             var tree = ParseTree(text, options);
             _node = tree.GetCompilationUnitRoot();
-            if (verifyErrors)
-            {
-                var actualErrors = _node.GetDiagnostics();
-                actualErrors.Verify(expectedErrors);
-            }
+            var actualErrors = _node.GetDiagnostics();
```

Comment:
For reviewers: this is the important part of the change. It's no longer possible to specify 'verifyErrors: false'.